spec/integration/ssl/autosign_spec.rb

```diff
@@ -12,10 +14,16 @@
         '1.3.6.1.4.1.34380.2.2' => # system IPs in hex
           [ 0xC0A80001, # 192.168.0.1
             0xC0A80101 ], # 192.168.1.1
+        # different UTF-8 widths
+        # 1-byte A
+        # 2-byte ۿ - http://www.fileformat.info/info/unicode/char/06ff/index.htm - 0xDB 0xBF / 219 191
+        # 3-byte ᚠ - http://www.fileformat.info/info/unicode/char/16A0/index.htm - 0xE1 0x9A 0xA0 / 225 154 160
+        # 4-byte 𠜎 - http://www.fileformat.info/info/unicode/char/2070E/index.htm - 0xF0 0xA0 0x9C 0x8E / 240 160 156 142
+        '1.2.840.113549.1.9.7' => utf8_value,
```

Comment:
Ah yeah, looked at that one, but missed what I *should* have changed there.

Should also add another line asserting that `1.2.840.113549.1.9.7` is not present in the certificate as well.